test/system/issue_creations_test.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require "application_system_test_case"
+
+class IssueCreationsTest < ApplicationSystemTestCase
+  test "visiting the index" do
```

Comment:
This test description doesn't describe what the test is looking for - my rule of thumb is to ask, if this test failed would the name tell you what was wrong? In this case, I'd suggest changing it to something like "Creating an issue on a train station is titled 'Report Train Issue'" or something like that.